src/components/post-card/post-card.module.scss

```diff
@@ -1,112 +1,229 @@
-		background-color: var(--surface_primary_emphasis-low);
-		border-color: var(--surface_primary_emphasis-high);
-	}
+.authorName {
+	margin: 0;
+	text-decoration: none;
+}
 
-	&:focus {
-		background-color: var(--background_focus);
-		border-color: var(--focus-outline_primary);
-	}
+.authorListContainer {
+	display: flex;
+	flex-direction: row;
+	gap: var(--card_detail-row_gap);
+	align-items: center;
 }
 
-.image {
-	position: absolute;
-	top: 0;
-	left: 50%;
-	transform: translateX(-50%);
+.cardIcon {
+	padding: var(--icon-size-dense-padding);
+	color: var(--card_detail-row_icon_color);
+	height: var(--card_detail-row_icon_size);
+	width: var(--card_detail-row_icon_size);
+}
 
+.cardIcon > svg {
 	height: 100%;
-	opacity: 0.5;
+	width: 100%;
+}
 
-	z-index: -1;
-	mask-image: linear-gradient(to bottom, #000F, #0000);
-	-webkit-mask-image: linear-gradient(to bottom, #000F, #0000);
-	background-color: #FFF;
+.dateAndWordCount {
+	display: flex;
+	flex-direction: row;
+	gap: var(--card_detail-row_gap);
+	align-items: center;
+}
 
-	@include darkTheme() {
-		filter: invert(1);
-		background-color: #000;
-	}
+.publishedDate {
+	margin: 0;
+	color: var(--card_detail-row_details_color);
 }
 
-.meta {
-	margin: var(--spc-4x) 0;
+.separatorDot {
+	margin: 0;
+	color: var(--card_detail-row_dot_color);
+}
 
-	&> * {
-		margin: 0;
-		line-height: 1;
-	}
+.wordCount {
+	margin: 0;
+	color: var(--card_detail-row_details_color);
+}
 
-	.date {
-		color: var(--foreground_emphasis-medium);
-	}
+.description {
+	margin: 0;
+	color: var(--card_snippet_color);
 }
 
-.header {
-	display: block;
-	text-decoration: none;
-	color: var(--foreground_emphasis-high);
+.cardList {
+	list-style: none;
+	margin: 0;
+	padding: 0;
+	display: flex;
+	flex-direction: row;
+	gap: var(--card_tags_gap);
+	margin-top: var(--card_tags_padding-top);
+}
 
-	& h2 {
-		margin: 0;
-	}
+.spacer {
+	height: 1px;
+	flex-grow: 1;
 }
 
-.excerpt {
-	margin: 0;
-	margin-bottom: var(--spc-4x);
+.extendedPostContainer,
+.regularPostContainer {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+}
+
+.extendedPostImageContainer {
+	height: 1px;
+	flex-grow: 1;
+	width: 100%;
+	position: relative;
+	mask-image: linear-gradient(180deg, rgba(0, 0, 0, 1), transparent 100%);
+	-webkit-mask-image: -webkit-gradient(
+		linear,
+		left top,
+		left bottom,
+		from(rgba(0, 0, 0, 1)),
+		to(transparent)
+	);
+	-webkit-mask-image: linear-gradient(
+		180deg,
+		rgba(0, 0, 0, 1),
+		transparent 100%
+	);
+	mask-image: -webkit-gradient(
+		linear,
+		left top,
+		left bottom,
+		from(rgba(0, 0, 0, 1)),
+		to(transparent)
+	);
+	mask-image: linear-gradient(180deg, rgba(0, 0, 0, 1), transparent 100%);
+}
+
+.extendedPostImage {
+	/* TODO: Change to `overlay` when background of image is black */
+	mix-blend-mode: soft-light;
+	height: 100%;
+	width: 100%;
+	position: absolute;
+	top: 0;
+	left: 0;
+	object-fit: cover;
+}
+
+.extendedPostImageBackground {
+	background-color: var(--primary_variant);
```

Comment:
I think this background can be placed on `.extendedPostImageContainer` for the same result to save a div :)

(I did test this in-browser first)